app/main/views/frameworks.py

```diff
@@ -782,23 +782,13 @@ def framework_updates_email_clarification_question(framework_slug):
             'clarification_question_text': clarification_question,
         }
 
-        try:
-            notify_client.send_email(
-                current_user.email_address,
-                template_name_or_id=current_app.config['NOTIFY_TEMPLATES']['confirmation_of_clarification_question'],
-                personalisation=confirmation_email_personalisation,
-                reference='clarification-question-confirm-{}'.format(hash_string(current_user.email_address))
-            )
-
-        except EmailError as e:
-            current_app.logger.error(
-                "{code}: Clarification question confirm email for email_hash {email_hash} failed to send. "
-                "Error: {error}",
-                extra={
-                    'error': str(e),
-                    'email_hash': hash_string(current_user.email_address),
-                    'code': 'clarification-question-confirm-email.fail'
-                })
+        notify_client.send_email(
+            current_user.email_address,
+            template_name_or_id=notify_client.templates['confirmation_of_clarification_question'],
+            personalisation=confirmation_email_personalisation,
+            reference='clarification-question-confirm-{}'.format(hash_string(current_user.email_address)),
+            reply_to_address_id=current_app.config['DM_ENQUIRIES_EMAIL_ADDRESS_UUID']
+        )
```

Comment:
Ah - I think we wanted this email to fail silently, hence the try..except. It's just a confirmation receipt so not crucial. There's a comment on line 775 for this.

Reply:
I think this will still fail silently if the email doesn't send, with the error being picked up by the client. If there's a `KeyError` then it'll fail more noisily. Is that okay?

Reply:
Hmm I think the client will still raise an `EmailError` (https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/email/dm_notify.py#L178). It'll handle the logging itself but the user will still get a 500.

Reply:
Ah, you're right. I'll put in the try...except